packages/app-mobile/components/screens/Note.tsx

```diff
@@ -835,7 +844,11 @@ class NoteScreenComponent extends BaseScreenComponent<Props, State> implements B
 			newNote.body += `\n${resourceTag}`;
 		}
 
-		this.setState({ note: newNote });
+		if (this.props.navigation.state.isNewPhotoNote) {
+			this.setState({ note: { ...newNote, title: 'Untitled' } });
```

Comment:
Why do you title the note as "Untitled"? And that should be localised anyway `_(....)`

Reply:
> Why do you title the note as "Untitled"? 

I made it so because, the photo's link becomes the title note (eg. ./758b410.....4d966).
Is there any other workaround for this?

Reply:
After taking the photo, it would IMHO be best from the user's point of view if the cursor were placed directly on the _empty_ title field, so that a title could be entered immediately (with the keyboard already open). This would minimize the number of clicks needed to create a photo note, including its required title.

See also https://github.com/laurent22/joplin/issues/10237

Reply:
That's a very good idea. @Wr4th100 would you be able to implement it that way? So no title by default and cursor on title field. Because indeed I can't think of a good default title (maybe the date but not sure that would be better than just an empty field)

Reply:
Some app (can't remember which one) in the same scenario named the note something like "Photo taken on 2024-04-04 12:00:00"

Reply:
> That's a very good idea. @Wr4th100 would you be able to implement it that way? So no title by default and cursor on title field. Because indeed I can't think of a good default title (maybe the date but not sure that would be better than just an empty field)

Yeah, sure will look into it!

Reply:
@Wr4th100, have you been able to check this?

Reply:
> @Wr4th100, have you been able to check this?

Hey, apologies for the late reply. I have made the required changes but there is one issue.

The title text when left empty gets filled by the image URL. So, I have made the photo note title to be display the date as shown in the video below.

https://github.com/laurent22/joplin/assets/68044488/89cd89c6-8b4e-4dea-9329-fcc430d5aa7a

> See also #10237

I looked into this and found out that it is pretty easy to implement by just focusing the ref on render. But there is a very big delay from the creation of the note and the focusing of title text input. Maybe is it because it is in development mode? Or am I missing something?